src/Xamarin.Android.Build.Tasks/Tasks/AndroidSignPackage.cs

```diff
@@ -113,15 +111,7 @@ protected override void LogEventsFromTextOutput (string singleLine, MessageImpor
 			if (singleLine.Length == 0)
 				return;
 
-			if (singleLine.StartsWith ("Warning:")) {
-				hasWarnings = true;
-				return;
-			}
-
-			if (hasWarnings)
-				Log.LogCodedWarning (DefaultErrorCode, GenerateFullPathToTool (), 0, singleLine);
-			else
-				Log.LogMessage (singleLine, importance);
+			Log.LogMessage (singleLine, importance);
```

Comment:
Hmm. What other warnings can JarSigner produce? This will ignore them all.

Reply:
I think I found them here:
* String resources: http://hg.openjdk.java.net/jdk8u/jdk8u-dev/jdk/file/0fc878b99541/src/share/classes/sun/security/tools/jarsigner/Resources.java
* `Main.java`: http://hg.openjdk.java.net/jdk8u/jdk8u-dev/jdk/file/0fc878b99541/src/share/classes/sun/security/tools/jarsigner/Main.java

I really only want to ignore this message:
```
Warning:
The signer's certificate is self-signed.
No -tsa or -tsacert is provided and this jar is not timestamped. Without a timestamp, users may not be able to validate this jar after the signer certificate's expiration date (2049-06-13) or after any future revocation date.
```
Which is output on 3 lines!

Should I add a list of strings to ignore and only look for the words:
* `self-signed`
* `No -tsa or -tsacert`